Makefile

```diff
@@ -351,6 +360,19 @@ envtest: $(ENVTEST) ## Download envtest-setup locally if necessary.
 $(ENVTEST): $(LOCALBIN)
 	test -s $(LOCALBIN)/setup-envtest || GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
 
+.PHONY: yq
+yq: $(YQ)
```

Comment:
no need of this. yq is present in the builder image

Reply:
Maybe we can keep it for being able to execute `make gen` locally without installing `yq` manually into our PC. Just to add another test if `yq` is already installed and its version corresponds to the [builder image's one](https://github.com/istio/tools/blob/master/docker/build-tools/Dockerfile#L77). 
WDYT?